js/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
```

Comment:
empty package.json , if not required now, lets remove

Reply:
Sure